catalog-info.yaml

```diff
@@ -0,0 +1,19 @@
+# Documentation: https://www.notion.so/snyk/Entity-Metadata-1e313ca03f954a6680b2b6b7b3d95297
+# Validation Tool: https://github.com/snyk/prodsec-github-bot
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: snyk-iac-parsers
+  annotations:
+    github.com/project-slug: snyk/snyk-iac-parsers
+    circleci.com/project-slug: github/snyk/snyk-iac-parsers
+  description: <
+    A Golang library that exposes functions for parsing YAML, HCL2, Terraform (including variable dereferencing) and Terraform Plan (JSON) files.
+  labels:
+    snyk.io/businessCriticality: medium
+    snyk.io/visibility: public
+    snyk.io/metadata-version: "2022-23-01"
+spec:
+  type: library
+  lifecycle: production
+  owner: "iac-language-support"
```

Comment:
For the moment can we make this `group-infrastructure-as-code` as the handle needs to align with one of the R&D groups.